dimagi/pages/templates/pages/pillar/mobile_data_collection.html

```diff
@@ -1,11 +1,15 @@
-{% extends "base.html" %}
+{% extends "amp/base.amp.html" %}
```

Comment:
hmm, so aren't AMP pages a separate experience specifically just for mobile? 

Changing this line also affects the desktop experience, and we need to figure out a different way of doing this. If you are using two separate templates, then user agent detection should happen on the server side and serve the appropriate base template.